doc/source/serve/production.md

```diff
@@ -352,7 +352,9 @@ Check for this variable in your environment to make sure you're using your desir
 
 The Serve CLI offers two commands to help you inspect your Serve application in production: `serve config` and `serve status`.
 
-If you're working with a remote cluster, `serve config` and `serve status` also offer an `--address/-a` argument to access your cluster. Check out [the previous section](serve-in-production-remote-cluster) for more info on this argument.
+If you're working with a remote cluster, `serve config` and `serve status` also offer an `--address/-a` argument to access your cluster.
+
+By default, all the Serve CLI commands assume that you're working with a local cluster, so if you don't specify an `--address/-a` value, they use the Ray agent address associated with a local cluster started by `ray start --head`. However, if the `RAY_AGENT_ADDRESS` environment variable is set, all Serve CLI commands will default to that value instead (unless you also specify an `--address/-a` value).
```

Comment:
Can we include a link to the `(serve-in-production-remote-cluster)` section somewhere in this paragraph, so users can refer to the extensive explanation?